Env/LogTraining.py

```diff
@@ -70,14 +73,36 @@ def _on_step(self) -> bool:
         :return: (bool) If the callback returns False, training is aborted early.
         """
 
+        # Now the length of the info_dict entries is know, so we update the headers accordingly
```

Comment:
typo: is knowN

Reply:
Bin mitten im Trainingsprozess gerade, update ich später. Danke!